cmd/kubernetesDeploy.go

```diff
@@ -16,6 +17,7 @@ import (
 	"github.com/SAP/jenkins-library/pkg/kubernetes"
 	"github.com/SAP/jenkins-library/pkg/log"
 	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/pkg/errors"
```

Comment:
We should use the native "errors" package instead

Reply:
Based on the [DEVELOPMENT.md](https://github.com/SAP/jenkins-library/blob/master/DEVELOPMENT.md#error-handling) section about the error handling, the `github.com/pkg/errors` package is chosen for wrapping errors within Piper

Reply:
Ok, my fault.